setup.sh

```diff
@@ -24,7 +24,9 @@ COMPOSER_YML_URL="http://kuzzle.io/docker-compose.yml"
 SUPPORT_MAIL="support@kuzzle.io"
 KUZZLE_DIR="${HOME}/.kuzzle"
 COMPOSER_YML_PATH="${KUZZLE_DIR}/docker-compose.yml"
+README_PATH="${KUZZLE_DIR}/README"
 FIRST_INSTALL=0
+HAS_GROUP_DOCKER="$(groups | grep docker)"
```

Comment:
nitpicking: this does not guarantee the user is allowed to run docker.
If he's just added himself to the docker group without login out and in again, this variable will be set while the user has no permission yet.

Reply:
True. Any suggestions?

Reply:
Something like that?

```
docker info > /dev/null 2>&1 || echo 'could not run docker info. Permission issue?'
```
or, if you want to check on the error message
```
docker_out=$(docker info 2>&1)
if [[ $? > 0 ]] && [[ "$docker_out" ~= "^Got permission denied" ]]; then
.. 
fi
```

Reply:
neat!